client/chat/channel.tsx

```diff
@@ -322,45 +335,42 @@ function renderMessage(msg: Message) {
   }
 }
 
-type UserEntry = [userId: SbUserId, username: string | undefined]
+type UserEntry = Map<SbUserId, string | undefined>
 
 function useUserEntriesSelector(userIds: ReadonlySet<SbUserId> | undefined) {
   return useCallback(
-    (state: RootState): ReadonlyArray<UserEntry> => {
+    (state: RootState): UserEntry => {
```

Comment:
The thing you return here should be readonly (probably deeply, so you can just use `Immutable<>`)